src/blockrelay/graphene_set.cpp

```diff
@@ -254,7 +254,7 @@ std::vector<unsigned char> CGrapheneSet::EncodeRank(std::vector<uint64_t> items,
         assert(ceil(log2(item)) <= nBitsPerItem);
 
         for (uint16_t j = 0; j < nBitsPerItem; j++)
-            bits[j + i * nBitsPerItem] = (item >> j) & 1;
+            bits[j + i * nBitsPerItem] = static_cast<bool>((item >> j) & 1);
```

Comment:
I feel this kind of extra code obfuscates more than it helps. Is there a way to fine tune the warning flags of your compiler to suppress this particular type of warning?